resharper/resharper-unity/src/Rider/CodeInsights/AbstractUnityImplicitProvider.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using JetBrains.Application.UI.Controls.GotoByName;
+using JetBrains.Application.UI.PopupLayout;
+using JetBrains.ProjectModel;
+using JetBrains.ReSharper.Daemon.CodeInsights;
+using JetBrains.ReSharper.Host.Features.TextControls;
+using JetBrains.ReSharper.Plugins.Unity.Resources;
+using JetBrains.ReSharper.Psi;
+using JetBrains.Rider.Model;
+using JetBrains.TextControl.TextControlsManagement;
+using JetBrains.UI.Icons;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Rider.CodeInsights
+{
+    public abstract class AbstractUnityImplicitProvider : ICodeInsightsProvider
+    {
+        public IconId IconId => LogoThemedIcons.UnityLogo.Id;
```

Comment:
We need a new icon here, that fits with the style for the other code insight providers. Can you add a new YouTrack request, or add a comment to [RIDER-21140](https://youtrack.jetbrains.com/issue/RIDER-21140), please?